### ISSUE TITLE
Create gtz recipe from binary file

### ISSUE THREAD

**Issue by tizzir** [bioconda/bioconda-recipes#7690]
Hello, I'm looking into creating a recipe for [gtz](https://github.com/Genetalks/gtz), but, it's packaged rather strangely.

Essentially it comes in the form of `tgz` files for different distros, each containing the application's binary, a folder with the required libraries, and a script you use to launch the application and put the libraries on the `PATH`.

Ideally, I think the recipe would ensure that all the libraries that come packaged are already on the system, and then put the binary alone in `/bin/`, and discard everything else.

However, I'm not sure if this is allowed based on the repo's [seemingly custom license](https://github.com/Genetalks/gtz/blob/master/License.md).

Is making a recipe for gtz even feasible when it's in this condition?

**Comment by colinbrislawn** [comment]
Looks like gtz uses a [BSD-2 clause license](https://opensource.org/licenses/BSD-2-Clause), although it would be best to have @superligen verify that.

As for the required libraries, it's best to list them as requirements, then have conda install them automatically before trying to build this software. If some are only needed on a specific system, you can flag that during the build:

```
requirements:
  build:
    - gcc # [linux]
    - llvm # [osx]
    - argtable2
```
This way linux and osx can get different pieces of software installed by conda, but `argtable2` is always installed. Something similar should be possible with gtz, and maybe the developer can help!

**Comment by tizzir** [comment]
Thank you @colinbrislawn! You're right, it does seem pretty similar to BSD-2, just without permitting redistribution *with* modification. But I'll reach out to @superligen to confirm and see if it's okay to turn it into a recipe.

I've created a recipe locally just to test compatibility, and while `conda build` can build it fine, I'm running into a bit of trouble with Circle CI; namely that GTZ at runtime will complain that `/lib64/libc.so.6` does not exist or is not `version 'GLIBC_2.14'`. Am I correct to assume that libc6 is not the sort of package that would be (or should be) found in a conda channel?

**Comment by colinbrislawn** [comment]
Yes, let's sort out that license! Modified FOSS licences are very misleading.

I'm pretty sure you can put anything into a conda package. Let's see what other folks recommend about this c compiler  @druvus @bgruening

**Comment by tizzir** [comment]
Good news, @superligen responded to the inquiry and modified the GTZ license to permit us to package it for Bioconda!

I'm wondering, though, are there going to be issues unless I use the specific commit that has the license change? There are no releases with this new license yet. Or do license alterations retroactively apply to previous commits?

**Comment by superligen** [comment]
Hi,      We confirm that this license alterations retroactively apply to previous commits.     Thank you.     Genetalks,     Gen Li ------------------------------------------------------------------发件人：Matthew Spelchak <notifications@github.com>发送时间：2018年2月16日(星期五) 21:29收件人：bioconda/bioconda-recipes <bioconda-recipes@noreply.github.com>抄　送：李根 <gen.li@genetalks.com>; Mention <mention@noreply.github.com>主　题：Re: [bioconda/bioconda-recipes] Create gtz recipe from binary file (#7690)
Good news, @superligen responded to the inquiry and modified the GTZ license to permit us to package it for Bioconda!I'm wondering, though, are there going to be issues unless I use the specific commit that has the license change? There are no releases with this new license yet. Or do license alterations retroactively apply to previous commits?—

**Comment by tizzir** [comment]
Thanks @superligen for the confirmation!

I'll go ahead creating a glibc conda recipe for GTZ to rely on unless I hear otherwise.
